redirects-transition.conf

```diff
@@ -83,7 +83,8 @@ rewrite ^/disclosureie/ienational.do?candOffice=S https://www.fec.gov/data/indep
 rewrite ^/disclosurep/pnational.do https://www.fec.gov/data/candidates/president/presidential-map/ redirect;
 
 # eeo/ redirects
-rewrite ^/eeo/eeo.shtml https://www.fec.gov/about/equal-employment-opportunity/ redirect; 
+rewrite ^/eeo/eeo.shtml https://www.fec.gov/about/equal-employment-opportunity/ redirect;
+rewrite ^/eeo/policy_statement.shtml https://www.fec.gov/about/equal-employment-opportunity/ redirect;
```

Comment:
@dorothyyeager Should we be more specific with the EEO policy statement redirect to go to the anchor here: https://www.fec.gov/about/equal-employment-opportunity/#policies-and-statements

Reply:
I think so. I'll update my PR.